helm/hpcc/values.yaml

```diff
@@ -493,7 +497,7 @@ esp:
 - name: eclwatch
   ## Pre-configured esp applications include eclwatch, eclservices, and eclqueries
   application: eclwatch
-  auth: none
+  auth: test
```

Comment:
Any other opinion?

Reply:
IMHO, it will be frustrating to first time users that download and install, then try to load ECLWatch.  They will be forced to enter a username/pwd, which they have to retrieve from the docs (and possibly provides a false sense of security).  Just one more frustrating step in the learning curve, and I am not sure how it benefits a user that isn't a tester

Reply:
I probably wouldn't make it the default, unless @richardkchapman or @ghalliday disagree.  I know there was a discussion similar to making it the default at some point.  But we would need to make sure the user knew what to put in as the username if we did.

Reply:
I think it would make sense for the startall.sh to default it to test.  I think we should have a separate developer.yaml file which is used by the startall.sh to ensure certain components are configured how we would expect.  A separate PR though.

Reply:
Changing to none